src/manual/using/actions/spec.md

```diff
@@ -281,6 +281,39 @@ Note that the result of rendering `hookPayload` becomes the context for another
 JSON-e rendering performed by the hooks service.  Do not confuse the two
 operations!
 
+### Choosing a Kind
```

Comment:
Some of this belong more in action design guidelines than the action specification.

This is a nit, but it would be nice to corner these discussion topics and examples into a separate document.
But I'm okay with this here too :)

Reply:
I can move it to a `guidelines.md`..